ring_doorbell/doorbot.py

```diff
@@ -214,14 +213,17 @@ def existing_doorbell_type_enabled(self) -> bool | None:
             return self._get_chime_setting("enable")
         return False
 
-    async def async_set_existing_doorbell_type_enabled(self, value: bool) -> None:  # noqa: FBT001
-        """Enable/disable the existing doorbell if Digital/Mechanical."""
+    async def async_set_existing_doorbell_type_enabled(self, value: int) -> None:
+        """Enable/disable the existing doorbell if Digital/Mechanical.
+        0: Off
+        1: On
+        """
+
```

Comment:
```suggestion
    async def async_set_existing_doorbell_type_enabled(self, enable: bool) -> None: # noqa: FBT001
        """Enable/disable the existing doorbell if Digital/Mechanical."""
	value = 1 if enable else 0
```
We should maintain the `bool` `in the public interface and convert it internally to the value the api expects.

Reply:
Understood and thank you for the quick response! Will get to work on those changes and make sure to get poetry in place.

Reply:
Thanks! Took this one through the proper steps. I also validated the change against my test script, this time using `await doorbell.async_set_existing_doorbell_type_enabled(False)` and `await doorbell.async_set_existing_doorbell_type_enabled(True)` to flip back and forth and it worked as expected, including adding the bool check logic back in,